go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/UpCloudLtd/upcloud-go-api/v6
 
-go 1.18
```

Comment:
Should we still keep this as minimum supported Go version 1.19?

Reply:
Thanks, good point! Version 1.21 is not far from being released but at least switch to a supported version for now 👍 I'll update the PR

Reply:
Done!